src/components/Profile/ProfileBlurb.tsx

```diff
@@ -1,18 +1,20 @@
-import React from 'react'
+import React, { ReactNode } from 'react'
+import { Flex, Text } from 'rebass/styled-components'
 
 import Twemoji from 'components/Icons/Twemoji'
+import ProfileHeading from 'components/Profile/ProfileHeading'
+import ProfileStyledLink from 'components/Profile/ProfileStyledLink'
 
-import ProfileHeading from './ProfileHeading'
-import ProfileStyledLink from './ProfileStyledLink'
-
-const BEIGBEDER_URL = 'http://jstu.art/ojVB'
-const COLING_URL = 'http://jstu.art/oijk'
-const USPO_URL = 'http://jstu.art/oj1g'
+const BEIGBEDER_URL = 'https://jstu.art/ojVB'
+const COLING_URL = 'https://jstu.art/oijk'
+const USPO_URL = 'https://jstu.art/oj1g'
 
 const {
+  Camera,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)